### PR TITLE
Cant read NPY files created on win64 python2

### PR DESCRIPTION
**Pull request by roblovelock**
Some NPY files created using python2 on windows can't be read as an 'L' is added to the array shape

Fix for: https://github.com/numpy/numpy/issues/5170
Replicates: https://github.com/numpy/numpy/pull/5178

### CODE REVIEW

File: src/main/kotlin/org/jetbrains/bio/npy/DictParser.kt [hunk 1]
```diff
@@ -41,7 +41,7 @@ private fun parseTuple(lexer: PeekingIterator<SpannedToken>): List<Int> {
     val acc = ArrayList<Int>()
     while (lexer.peek().token != Token.RPAR) {
         val item = lexer.eat(Token.INT)
-        acc.add(item.span.toInt())
+        acc.add(item.span.filterNot { c -> c == 'L' }.toInt())
```
Comment (superbobry) [inline src/main/kotlin/org/jetbrains/bio/npy/DictParser.kt:44]: Could you change this to `substringBeforeLast('L')`? It is operationally identical to your version, but illustrates the fact that we only expect L in the end.

**Comment by superbobry** [comment]
Good catch, thanks! We've indeed missed the int/long separation on Python2.

**Comment by superbobry** [comment]
I've decided to merge this as, thank you!

**Comment by roblovelock** [comment]
Great, thanks. If you want me to raise a new pull request with the change to substringBeforeLast('L') let me know

**Comment by roblovelock** [comment]
Just seen you have already made the substringBeforeLast('L') changes. Thanks for merging this

**Pull request merged**